app/javascript/dashboard/routes/dashboard/settings/profile/Index.vue

```diff
@@ -95,6 +97,9 @@ export default {
       currentUserId: 'getCurrentUserID',
       globalConfig: 'globalConfig/get',
     }),
+    isMfaEnabled() {
+      return window.chatwootConfig?.isMfaEnabled === 'true';
```

Comment:
Avoid this string comparison for boolean variables. There is a helper `parseBoolean` ref:  https://github.com/chatwoot/chatwoot/blob/cc21016e6d60488c31162c2e399dd929fdb25c0e/app/javascript/v3/views/login/Index.vue#L82